activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -53,7 +64,14 @@ def self.initialize_i18n(app)
       # Restore available locales check so it will take place from now on.
       I18n.enforce_available_locales = enforce_available_locales
 
-      reloader = ActiveSupport::FileUpdateChecker.new(I18n.load_path.dup){ I18n.reload! }
+      directories = Helpers.watched_dirs_with_extensions(reloadable_paths)
```

Comment:
Hm, `watched` feels off to me, as the directories haven't been watched yet. What do you think about `extensions_by_absolute_paths_for`?